manifests/filter/header_user.pp

```diff
@@ -0,0 +1,66 @@
+# == Resource: repose::filter::header_user
+#
+# This is a resource for generating header-user configuration files
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool. Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String. Config filename
+# Defaults to <tt>header-user.cfg.xml</tt>
+#
+# [*source-headers*]
+# List containing source headers with the ids X-Auth-Token and X-Forwarded-For.
+# X-Auth-Token defaults to <tt>.95</tt>
+# X-Forwarded-For defaults to <tt>0.5</tt>
```

Comment:
What are these referencing?

Reply:
ah, from the initial commit when this was a static page. removing